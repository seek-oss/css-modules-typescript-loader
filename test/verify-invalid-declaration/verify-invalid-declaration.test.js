const compiler = require('../compiler.js');

test('Can error on invalid declaration', async () => {
  await expect(
    compiler(require.resolve('./index.js'), {
      mode: 'verify'
    })
  ).rejects.toMatchObject({
    failed: true
  });
});
