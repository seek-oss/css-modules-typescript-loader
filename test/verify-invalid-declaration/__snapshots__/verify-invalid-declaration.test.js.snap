// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Can error on invalid declaration 1`] = `
"Error: Generated type declaration file is outdated. Run webpack and commit the updated type declaration for '/test/verify-invalid-declaration/index.css.d.ts'

   // This file is automatically generated.
   // Please do not change this file!
   interface CssExports {
 -   'classInBothFiles': string;
 -   'classInTypeScriptFile': string;
 +   readonly 'classInBothFiles': string;
 +   readonly 'classInCssFile': string;
   }
   export const cssExports: CssExports;
   export default cssExports;
   

"
`;
